.travis.yml

```diff
@@ -2,20 +2,21 @@ language: c
 sudo: false
 services:
   - docker
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+install:
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+  - wget https://raw.githubusercontent.com/xapi-project/xapi-travis-scripts/master/coverage.sh
 script: bash -ex ./.travis-docker.sh
 env:
   global:
     - PINS="stdext:. xapi-stdext:. xapi-stdext-base64:. xapi-stdext-bigbuffer:. xapi-stdext-date:. xapi-stdext-deprecated:. xapi-stdext-encodings:. xapi-stdext-monadic:. xapi-stdext-pervasives:. xapi-stdext-range:. xapi-stdext-std:. xapi-stdext-threads:. xapi-stdext-unix:. xapi-stdext-zerocheck:."
-    - DISTRO="centos-7"
-    - TESTS=false
+    - DISTRO="ubuntu-16.04"
```

Comment:
I really would like to keep centos-7... We should rethink our whole CI to use `xs-opam:latest` as a base...

Reply:
It does not work because it was impossible to remove the fake aspcud in the docker image, even  POST_INSTALL_HOOK didn't work :(

Reply:
Yes, I understand. I am still wishful